app/assets/javascripts/app/books.coffee

```diff
@@ -0,0 +1,44 @@
+$(document).on "turbolinks:load", ->
+  $ ->
+    $('#read-more').click (event) ->
+      event.preventDefault()
+      $('#long-desc').toggle()
+      $('#short-desc').toggle()
+      return
+    return
+  $ ->
+    $('#decrease').click (event) ->
+      event.preventDefault()
+      amount_in_input = parseInt($('#order-quantity').val())
+      if (amount_in_input > 1)
+        $('#order-quantity').val(amount_in_input - 1)
+      return
+    return
+  $ ->
+    $('#increase').click (event) ->
+      event.preventDefault()
+      amount_in_input = parseInt($('#order-quantity').val())
+      current_amount = parseInt($('#quantity').val())
+      if (amount_in_input < current_amount)
+        $('#order-quantity').val(amount_in_input + 1)
+      return
+    return
+  $ ->
+    $('#review').on 'keyup', ->
+      length = $(this).val().length
+      $('#review-body-count').html 500 - length
+      return
+    return
+  $ ->
+    $('#order-quantity').on 'keyup', ->
+      if isNaN($('#order-quantity').val())
```

Comment:
better specify variable(smth like `order_quantity`) and use it.
Also have a look to http://coffeescript.org/#try:if%20not%20MyVariable%3F%0A%20%20MyVariable%20%3D%20%22assign%20a%20value%22